extensions/familysearch/familysearch-api-model/src/test/java/org/familysearch/platform/ct/FamilySearchFactTypeTest.java

```diff
@@ -31,6 +31,19 @@ public void testIt() {
     testType("http://familysearch.org/v1/TitleOfNobility", FamilySearchFactType.TitleOfNobility);
     testType("http://familysearch.org/v1/TribeName", FamilySearchFactType.TribeName);
 
+    // association fact types
+    //testType("http://familysearch.org/v1/Apprenticeship", FamilySearchFactType.Apprenticeship); // exists as a core Gedcomx FactType
```

Comment:
One issue here is for TF the Apprenticeship fact type is in the familysearch namespace
But it is in core Gedcomx and I believe that will use http://gedcomx.org/Apprenticeship